config/ols.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: ols-proxy
+  name: ols-proxy
+spec:
+  ports:
+  - name: https
+    port: 8443
+    targetPort: https
+  selector:
+    app: ols-proxy
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ols-proxy
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ols-proxy
+  template:
+    metadata:
+      labels:
+        app: ols-proxy
+    spec:
+      securityContext:
+        runAsUser: 65532
+      serviceAccountName: ols-proxy
+      containers:
+      - name: ols-proxy
+        image: quay.io/brancz/kube-rbac-proxy:v0.15.0
+        args:
+        - "--secure-listen-address=0.0.0.0:8443"
+        - "--upstream=http://0.0.0.0:8080"
+        - "--ignore-paths=/healthz"
+        - "--auth-token-audiences=ols-proxy.ols.svc"
+        - "--logtostderr=true"
+        - "--v=10"
+        ports:
+        - containerPort: 8443
+          name: https
+        securityContext:
+          allowPrivilegeEscalation: false
+      - name: ols
+        image: quay.io/vchalla/lightspeed-service:latest
```

Comment:
We will need to replace this with our official image repository if any.

Reply:
since you appear to be using kustomize, can you make this is parameter for now?

Reply:
Yes parameterized this image param, using envs